infrastructure/kubernetes/pagopa-proxy.yml

```diff
@@ -0,0 +1,87 @@
+#
+# The pagopa-proxy
+#
+---
+#
+# Deploy the pods
+#
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: pagopa-proxy
+  namespace: default
+  labels:
+    app: pagopa-proxy
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 50%
+      maxSurge: 1
+  revisionHistoryLimit: 10
+  template:
+    metadata:
+      name: pagopa-proxy
+      namespace: default
+      labels:
+        app: pagopa-proxy
+    spec:
+      containers:
+      - image: "teamdigitale/italia-pagopa-proxy:v0.2.0"
+        name: pagopa-proxy
+        ports:
+        - name: http-port
+          containerPort: 80
+        env:
+        - name: "WINSTON_LOG_LEVEL"
+          value: "debug"
+        - name: "PAGOPAPROXY_HOST"
+          value: "localhost"
+        - name: "PAGOPAPROXY_PORT"
+          value: "80"
+        - name: "PAGOPA_HOST"
+          value: "nd"
+        - name: "PAGOPA_PORT"
+          value: "80"
+        - name: "PAGOPA_TOKEN"
+          valueFrom:
+            secretKeyRef:
+              name: pagopa-proxy
+              key: pagopa_token
+        - name: "REDIS_DB_URL"
+          value: "agid-redis-dev.redis.cache.windows.net"
+        - name: "REDIS_DB_PORT"
+          value: "6380"
+        - name: "REDIS_DB_PASSWORD"
+          valueFrom:
+            secretKeyRef:
+              name: pagopa-proxy
+              key: redis_password
```

Comment:
@gunzip is it possible to create a new account on the same Redis instance used by the backend?